dialect/pgdialect/range.go

```diff
@@ -1,104 +1,202 @@
 package pgdialect
 
 import (
+	"bytes"
 	"database/sql"
 	"fmt"
-	"io"
 	"time"
 
 	"github.com/uptrace/bun/internal"
 	"github.com/uptrace/bun/schema"
 )
 
-type MultiRange[T any] []Range[T]
-
 type Range[T any] struct {
 	Lower, Upper           T
 	LowerBound, UpperBound RangeBound
 }
 
+type MultiRange[T any] []Range[T]
+
 type RangeBound byte
 
 const (
 	RangeBoundInclusiveLeft  RangeBound = '['
 	RangeBoundInclusiveRight RangeBound = ']'
 	RangeBoundExclusiveLeft  RangeBound = '('
 	RangeBoundExclusiveRight RangeBound = ')'
+	RangeBoundUnbound        RangeBound = 'U' // U is just a placeholder and has no actual meaning
```

Comment:
U is a surrogate bound that should not be appended to the query and which indicates unbound range

Reply:
U and E are just placeholders in Go.